bindings/tflite/java/com/google/iree/native/interpreter_wrapper_test.cc

```diff
@@ -0,0 +1,123 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "bindings/tflite/java/com/google/iree/native/interpreter_wrapper.h"
+
+#include <memory>
+
+#include "bindings/tflite/java/com/google/iree/native/model_wrapper.h"
+#include "bindings/tflite/java/com/google/iree/native/options_wrapper.h"
+#include "bindings/tflite/java/com/google/iree/native/tensor_wrapper.h"
+#include "iree/base/api.h"
+#include "iree/testing/gtest.h"
+
+// Test model is available both on the filesystem and here for embedding testing
+// embedding the module directly in a binary.
+#include "bindings/tflite/testdata/add_static.h"
+#define IREE_BINDINGS_TFLITE_TESTDATA_ADD_STATIC_EMBEDDED_DATA \
+  iree::bindings::tflite::testdata::add_static_create()->data
+#define IREE_BINDINGS_TFLITE_TESTDATA_ADD_STATIC_EMBEDDED_SIZE \
+  iree::bindings::tflite::testdata::add_static_create()->size
+
+#define ASSERT_OK(status) ASSERT_EQ(status, iree_ok_status())
```

Comment:
Can't you reuse https://github.com/google/iree/blob/b668ef8c84ae3a01642e1425fafccae66a1d5ce0/iree/testing/status_matchers.h#L327-L335 

? (there are implicit conversions from the C status to the C++ Status)

See sample usage here: https://github.com/google/iree/blob/b668ef8c84ae3a01642e1425fafccae66a1d5ce0/iree/base/internal/wait_handle_test.cc#L100-L106